src/main/java/Command.java

```diff
@@ -0,0 +1,187 @@
+                    task = new DeadlineTask(description.toString(),false, deadline.toString());
+                    tasks.addTask(task);
+                    ui.uiForAdd(tasks, task);
+                    storage.save(tasks);
+                } catch (DateTimeParseException e){
+                    System.err.println("Please use yyyy-mm-dd");
+                    throw new DukeException("Please use dd-MM-yyyy", DukeExceptionType.WRONG_TIME, Commands.DEADLINE);
+                }
+                break;
+            }
+            case "event" :{
+                Task task;
+                StringBuilder description = new StringBuilder();
+                StringBuilder time = new StringBuilder();
+                if (inputLine.length < 2) {
+                    throw new DukeException("", DukeExceptionType.NO_DESCRIPTION, Commands.EVENT);
+                }
+                int index = 1;
+                while(!inputLine[index].equals("/at")){
+                    description.append(inputLine[index]).append(" ");
+                    index++;
+                    if(index == inputLine.length){
+                        throw new DukeException("", DukeExceptionType.EMPTY_TIME, Commands.EVENT);
+                    }
+                }
+                if(description.length() < 1){
+                    throw new DukeException("", DukeExceptionType.NO_DESCRIPTION, Commands.EVENT);
+                }
+                index++;
+                while(index < inputLine.length){
+                    time.append(inputLine[index]).append(" ");
+                    index++;
+                }
+                if(time.length() < 1){
+                    throw new DukeException("", DukeExceptionType.EMPTY_TIME, Commands.EVENT);
+                }
+                time.deleteCharAt(time.length() - 1);
+                try{
+                    task = new EventTask(description.toString(),false, time.toString());
+                    tasks.addTask(task);
+                    ui.uiForAdd(tasks, task);
+                    storage.save(tasks);
+                } catch (DateTimeParseException e){
+                    System.err.println("Please use yyyy-mm-dd format for time");
+                    throw new DukeException("Please use dd-MM-yyyy HHmm format", DukeExceptionType.WRONG_TIME, Commands.EVENT);
+                }
+                break;
+            }
+            case "find" : {
+                if (inputLine.length < 2) {
+                    throw new DukeException("find needs keyword", DukeExceptionType.NO_DESCRIPTION, Commands.FIND);
+                } else {
+                    StringBuilder description = new StringBuilder();
+                    for (int i = 1; i < inputLine.length; i++) {
+                        description.append(inputLine[i]).append(" ");
+                    }
+                    List<Task> filteredList = tasks.search(description.toString());
+                    ui.uiForFind(new TaskList(filteredList));
+                }
```

Comment:
Perhaps you could include a "break;" after this line? I think this you missed this as you include break for every other cases.